_includes/head.html

```diff
@@ -0,0 +1,28 @@
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta charset="utf-8"/>
+  <title>
+    {% if page.title == null %}
+      {{ site.name }}
+    {% else %}
+      {{ page.title }} - {{ site.name }}
+    {% endif %}
+  </title>
+  <meta name="viewport" content="width=device-width"/>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/normalize.css"/>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/main.css"/>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/syntax.css"/>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/jquery-ui.min.css"/>
+  <link rel="shortcut icon" type="image/x-icon" href="{{ site.baseurl }}/favicon.ico?"/>
+
+<script src="{{ site.baseurl }}/assets/js/jquery-1.11.2.min.js"></script>
+<script src="{{ site.baseurl }}/assets/js/jquery-migrate-1.2.1.min.js"></script>
+<script src="{{ site.baseurl }}/assets/js/accordion.js"></script>
+<script src="{{ site.baseurl }}/assets/js/jquery-ui.min.js"></script>
+
+  <!--[if lt IE 9]>
+    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <script src="{{ site.baseurl }}/assets/js/respond.min.js"></script>
+  <![endif]-->
+
+</head>
```

Comment:
I moved the `<head>` material into its own `_include`, to simplify the `default.html` layout a bit.